src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/TickerTask.java

```diff
@@ -215,12 +341,28 @@ public void halt() {
 
     @ParametersAreNonnullByDefault
     public void queueMove(Location from, Location to) {
-        movingQueue.put(from, to);
+        // This collection is iterated over in a different thread. Need to lock it.
+        new Throwable().printStackTrace();
```

Comment:
Why are you printing a stacktrace here...?

Reply:
This is from debugging. I just forgot to remove it